model/test_results_storage.go

```diff
@@ -8,15 +8,13 @@ import "github.com/mongodb/anser/bsonutil"
 // The prefix field indicates the name of the "sub-bucket". The top level
 // bucket is accesible via the cedar.Environment interface.
 type TestResultsArtifactInfo struct {
-	Type    PailType        `bson:"type"`
-	Prefix  string          `bson:"prefix"`
-	Version int             `bson:"version"`
-	Tests   map[string]bool `bson:"tests,omitempty"`
+	Type    PailType `bson:"type"`
+	Prefix  string   `bson:"prefix"`
+	Version int      `bson:"version"`
```

Comment:
don't we need to save timestamps for when analysis runs ran?

Reply:
no I don't think its necessary per our discussion on Friday. We will just have a job to recalculate historical stats for a given range of dates.